mt-kahypar/partition/refinement/fm/global_rollback.cpp

```diff
@@ -0,0 +1,420 @@
+#include "global_rollback.h"
+
+#include "tbb/parallel_scan.h"
+#include "mt-kahypar/partition/metrics.h"
```

Comment:
add again sequential rollback, since we need it for n-level

Reply:
done